src/Webform.js

```diff
@@ -1474,11 +1486,24 @@ export default class Webform extends NestedDataComponent {
   executeSubmit(options) {
     this.submitted = true;
     this.submitting = true;
+    this.clearServerErrors();
```

Comment:
Why did you place clearServerErrors in executeSubmit and not inside submit? Isn't it possible that beforeSubmit() throws an Error and then executeSubmit won't be executed?

Reply:
If I understand correctly, server errors should only be cleaned up when we have sent submission on the server. We will then receive a response that will either return updated errors or no errors at all. In either case, you need to clean up old mistakes